packages/stratego-four/src/game/gameSaga.js

```diff
@@ -38,14 +38,14 @@ function* initializePlayer({ player }) {
   yield put(
     setCacheKey(
       'currentGame',
-      stratego4.methods.currentGame.cacheCall(player, ops)
+      stratego4.methods.currentGame.cacheCall(player, { ...ops })
```

Comment:
Apparently, the new provider mucks with this object so I have to create a new one so subsequent calls can still use it.